### PR TITLE
use new make file

### PR DESCRIPTION
**Pull request by h-bragg**
Some tidying up.

- Modified MakeFile
- Removed Dockerfile
- Updated Matrix
- Updated travis.yml
- Added shared standards library

### CODE REVIEW

**Comment by h-bragg** [comment]
@wpillar Pinging you on this, as it might be useful for the php5.5 stuff.

File: README.md [hunk 1]
```diff
@@ -2,29 +2,20 @@
...
-[lang]: https://secure.php.net
-[package]: https://packagist.org/packages/graze/queue
-[license]: https://github.com/graze/queue/blob/master/LICENSE
-
-<!-- Images -->
-[ico-license]: https://img.shields.io/packagist/l/graze/queue.svg
-[ico-package]: https://img.shields.io/packagist/v/graze/queue.svg
-[ico-build]: https://img.shields.io/travis/graze/queue/master.svg
-[ico-engine]: https://img.shields.io/badge/php-%3E%3D5.5-8892BF.svg
-
```
Comment (wpillar) [inline README.md]: Not a fan?

**Comment by wpillar** [comment]
@h-bragg I'll pick my PR up again once this is in master.

File: Makefile [hunk 2]
```diff
@@ -1,44 +1,62 @@
...
-	docker run -t --rm \
-	-v $$(pwd):/app \
-	-v ~/.composer:/root/composer \
-	-v ~/.ssh:/root/.ssh:ro \
-	composer/composer --ansi --no-interaction $*
+	${DOCKER} run -t --rm \
+        -v $$(pwd):/usr/src/app \
+        -v ~/.composer:/root/composer \
+        -v ~/.ssh:/root/.ssh:ro \
+        graze/composer --no-interaction --prefer-dist $* $(filter-out $@,$(MAKECMDGOALS))
```
Comment (sjparkinson) [inline Makefile]: Nice 😄 .

File: README.md (hunk 1 above)
Comment (h-bragg) [inline README.md]: its quite nice, but I was copy/pasting from another README, and it was more of a pain to split out the images than not to.

**Pull request merged**